docs/workflows/account-setup.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Account Setup
+sidebar_position: 2
+---
+
+# Account Setup
+
+## Sign up for a developer account
+
+To get started, sign up using the following link: https://www.beyondidentity.com/developers/signup. Completing the form will generate several things behind the scenes:
+
+1. You will be given a [tenant](../platform-overview/architecture), which serves as a root container for all your cloud components.
+2. Inside this tenant you will find a "Beyond Identity Admin" [realm](../platform-overview/architecture).
+3. This realm is populated with an admin identity linked to your email address and two applications: "Beyond Identity Admin Console" and "Beyond Identity Management API".
+4. A [Universal Passkey](../platform-overview/passkeys-and-devices/what-are-passkeys.md) will be created and bound to your identity.
+5. The passkey will be [stored](../platform-overview/passkeys-and-devices/how-are-keys-stored.md) on your device or browser.
+6. With the passkey, you will be immediately logged into your own Beyond Identity Admin Console.
```

Comment:
I wonder if it would be helpful to add a visual here (either something similar to that visual you created for the acme app after creating a tenant, or this https://developer-docs-git-account-setup-devx-1169-beyondidentity.vercel.app/docs/v1/platform-overview/architecture)

Reply:
Hmm yea I was thinking about that too. I didn't want to list the whole architecture tree because that seems like too much info, so that's what I link to if you tap on tenant or realm.

Reply:
Just added a diagram, let me know what you think!